### PR TITLE
Fix repository creation timeouts and configs

### PR DESCRIPTION
**Pull request by jbartece**

### CODE REVIEW

File: maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java [hunk 1]
```diff
@@ -113,8 +118,18 @@ private Indy init(String accessToken) {
                 authenticator = new OAuth20BearerTokenAuthenticator(accessToken);
             }
             try {
-                indy = new Indy(baseUrl, authenticator, new IndyFoloAdminClientModule(), new IndyFoloContentClientModule(),
-                        new IndyPromoteClientModule()).connect();
+                SiteConfig siteConfig = new SiteConfigBuilder("indy", baseUrl)
+                        .withRequestTimeoutSeconds(120) // probably not good to go beyond this.
```
Comment (matejonnet) [inline maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java]: We should make it configurable.

File: maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java [hunk 2]
```diff
@@ -199,7 +214,7 @@ private void setupBuildRepos(BuildExecution execution, Indy indy)
             // if the product-level storage repo (for in-progress product builds) doesn't exist, create it.
             if (!indy.stores().exists(StoreType.hosted, buildContentId)) {
                 HostedRepository buildArtifacts = new HostedRepository(buildContentId);
-                buildArtifacts.setAllowSnapshots(true);
+                buildArtifacts.setAllowSnapshots(false);
```
Comment (matejonnet) [inline maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java]: We should make it configurable.

**Comment by jdcasey** [comment]
@jbartece Oops, I forgot to port this over. If you want to close this, I can port it, make the changes @matejonnet  requested, and submit another PR.

**Comment by jbartece** [comment]
@jdcasey That's fine, I'll do it, when I have some free time ;-)

**Comment by jbartece** [comment]
retest this please

**Pull request merged**